imports/lib/utils.js

```diff
@@ -90,6 +90,43 @@ export function removeTrailingSlash (str) {
   return str.replace(/\/$/, '')
 }
 
+export function setIsNavigatingBack (navigatingBack = false) {
```

Comment:
These helper utils are here in case the underlying implementation for how this piece of application state is maintained changes

Reply:
Can we rename "prevPaths" to "navigationHistory" (which seems more explicit to me)

Reply:
and  rename the various helpers accordingly: i.e. `addToNavHistory(path)` instead of `addPreviousPath(prevPath)`, `removeFromNavHistory(path)`, etc.

Reply:
Will do, makes total sense 👍

Reply:
Done via 3a0b7c1